src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -404,6 +407,46 @@ class device {
         return 0;
     }
 
+    /* new KDS which supported CU subdevice */
+    int parseCUSubdevStat() const
+    {
+        using tokenizer = boost::tokenizer< boost::char_separator<char> >;
+        boost::char_separator<char> sep(" ");
+        std::vector<std::string> custat;
+        std::string errmsg;
+
+        pcidev::get_dev(m_idx)->sysfs_get("", "kds_custat_raw", errmsg, custat);
```

Comment:
With regards to the kds_custat_raw, who are the clients of this data?

I ask for, it might be better it this data was formatted in a way that is easier to parse.  Some ideas:
1. Key / value pairs (e.g., "name":"cu1", "base_address":"0x1234", etc.)
2. Using comma deliminators: (e.g., "cu1","0x1234","",0x4, etc.)

Reply:
The xbutil is the client of this data. Now xbutil need below CU inforamtion: index, name (consist of kernel_name and instance_name), address, status, usage.
I could use comma as separator. This is fixed.